packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -84,6 +84,13 @@ const setMaxFs = (sdp, level = QUALITY_LEVELS.HIGH) => {
   return replaceSdp;
 };
 
+
+const setMaxFsOnRemoteSdp = (sdp) => {
+  sdp = sdp.replace(/(\na=fmtp:(\d+).*level-asymmetry-allowed=1.*)/gi, `$1;x-google-start-bitrate=${StaticConfig.meetings.bandwidth.startBitrate}`);
```

Comment:
I'm not sure if it's the best approach to search for `level-asymmetry-allowed=1` I think we can set `x-google-start-bitrate` even if level asymmetry is not allowed (so value 0 or not present in the fmtp line). Maybe it would be better to search for `profile-level-id=` string?